.gitattributes

```diff
@@ -1,8 +1,6 @@
 *.postcard  binary
-provider/testdata/data/baked/** linguist-generated=true
-provider/testdata/data/json/** linguist-generated=true
-provider/testdata/data/cldr/** linguist-generated=true
-provider/testdata/data/icuexport/** linguist-generated=true
+provider/testdata/data/** linguist-generated=true
```

Comment:
Issue: We had these pointing at subdirectories because there are certain files we _don't_ want to hide away, like the fingerprint files

Reply:
They *are* generated though